src/dbally/views/structured.py

```diff
@@ -56,14 +57,27 @@ async def ask(
         Returns:
             The result of the query.
         """
-        iql_generator = self.get_iql_generator(llm)
+
         filter_list = self.list_filters()
+        few_shot_list = self.list_few_shot()
+        iql_generator = self.get_iql_generator(llm)
+
+        input_formatter = (
+            DefaultFewShotInputFormatter(
+                question=query,
+                filters=filter_list,
+                examples=[
```

Comment:
Examples of structured views should be provided in a similar way to freeform views (related to refactoring `examples()` method).